discovery_engine_core/tooling/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM debian:bookworm-slim
+
+WORKDIR /service
+
+COPY ./ ./
+
+RUN chmod +x ./backend
+
+EXPOSE 3000
+
+CMD ["./backend"]
```

🎨 new line at the end
```suggestion
CMD ["./backend"]

```